custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -7,12 +7,38 @@
 import pandas as pd
 
 
+class Conditions:
```

Comment:
Wondering if these should have more descriptive names.

Reply:
I added descriptions.  For this I like them matching the field names in the specificaiton

Reply:
can/should these be actual Enums?

Reply:
short answer, they _could_ but it might not be worth the time